File: chroma-league/src/main/java/net/booone/chromaleague/hud/PredefinedKeySets.java
package net.booone.chromaleague.hud;

import net.booone.razersdk.sdk.RzKey;
import net.booone.razersdk.sdk.RzKeySelector;

import java.util.List;

import static net.booone.razersdk.sdk.RzKey.RZKEY_PAGEDOWN;
import static net.booone.razersdk.sdk.RzKey.RZKEY_PRINTSCREEN;

public final class PredefinedKeySets {
    public static final List<RzKey> BLACKWIDOW_FIRST_ROW = new RzKeySelector().withRow(0).sortedByColumn().asList();
    public static final List<RzKey> BLACKWIDOW_SECOND_ROW = new RzKeySelector().withRow(1).sortedByColumn().asList();
    public static final List<RzKey> BLACKWIDOW_THIRD_ROW = new RzKeySelector().withRow(2).sortedByColumn().asList();
    public static final List<RzKey> BLACKWIDOW_FOURTH_ROW = new RzKeySelector().withRow(3).sortedByColumn().asList();
    public static final List<RzKey> BLACKWIDOW_FIFTH_ROW = new RzKeySelector().withRow(4).sortedByColumn().asList();
    public static final List<RzKey> BLACKWIDOW_SIXTH_ROW = new RzKeySelector().withRow(5).sortedByColumn().asList();
    public static final List<RzKey> BLACKWIDOW_FUNCTIONAL = new RzKeySelector().withRectangleBetween(RZKEY_PRINTSCREEN, RZKEY_PAGEDOWN).asList();
    public static final List<RzKey> FIRST_NUMPAD_COLUMN = new RzKeySelector().withColumn(18).sortedByRow().asList();
    public static final List<RzKey> SECOND_NUMPAD_COLUMN = new RzKeySelector().withColumn(19).sortedByRow().asList();
    public static final List<RzKey> THIRD_NUMPAD_COLUMN = new RzKeySelector().withColumn(20).sortedByRow().asList();
    public static final List<RzKey> FOURTH_NUMPAD_COLUMN = new RzKeySelector().withColumn(21).sortedByRow().asList();

    private PredefinedKeySets() {
    }
}

File: src/hud/PredefinedKeySets.ts
import {RzKey} from '../razer-sdk/sdk/RzKey.js';
import {RzKeySelector} from '../razer-sdk/sdk/RzKeySelector.js';

export const BLACKWIDOW_FIRST_ROW: RzKey[] = new RzKeySelector().withRow(0).sortedByColumn().asList();
export const BLACKWIDOW_SECOND_ROW: RzKey[] = new RzKeySelector().withRow(1).sortedByColumn().asList();
export const BLACKWIDOW_THIRD_ROW: RzKey[] = new RzKeySelector().withRow(2).sortedByColumn().asList();
export const BLACKWIDOW_FOURTH_ROW: RzKey[] = new RzKeySelector().withRow(3).sortedByColumn().asList();
export const BLACKWIDOW_FIFTH_ROW: RzKey[] = new RzKeySelector().withRow(4).sortedByColumn().asList();
export const BLACKWIDOW_SIXTH_ROW: RzKey[] = new RzKeySelector().withRow(5).sortedByColumn().asList();
export const BLACKWIDOW_FUNCTIONAL: RzKey[] = new RzKeySelector().withRectangleBetween(RzKey.RZKEY_PRINTSCREEN, RzKey.RZKEY_PAGEDOWN).asList();
export const FIRST_NUMPAD_COLUMN: RzKey[] = new RzKeySelector().withColumn(18).sortedByRow().asList();
export const SECOND_NUMPAD_COLUMN: RzKey[] = new RzKeySelector().withColumn(19).sortedByRow().asList();
export const THIRD_NUMPAD_COLUMN: RzKey[] = new RzKeySelector().withColumn(20).sortedByRow().asList();
export const FOURTH_NUMPAD_COLUMN: RzKey[] = new RzKeySelector().withColumn(21).sortedByRow().asList();
